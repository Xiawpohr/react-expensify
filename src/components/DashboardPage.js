import React from 'react'

function DashboardPage (props) {
  return (
    <div>
      Dashboard page
    </div>
  )
}

export default DashboardPage
