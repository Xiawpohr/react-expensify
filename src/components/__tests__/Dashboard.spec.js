import React from 'react'
import { shallow } from 'enzyme'
import DashboardPage from '../DashboardPage.js'

test('should render Dashboard page correctly', () => {
  const wrapper = shallow(<DashboardPage />)
  expect(wrapper).toMatchSnapshot()
})
