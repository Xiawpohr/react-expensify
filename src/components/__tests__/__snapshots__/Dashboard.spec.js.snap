// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard page correctly 1`] = `
<div>
  <WithTheme(Connect(Component)) />
  <Connect(ExpenseListFilter) />
  <Connect(Component) />
</div>
`;
